Source/CarthageKit/Resolver.swift

```diff
@@ -163,9 +164,10 @@ public struct Resolver {
 	/// the specified node (or as a root otherwise).
 	///
 	/// This is a helper method, and not meant to be called from outside.
-	private func graphs(for nodes: [DependencyNode], dependencyOf: DependencyNode?, basedOnGraph inputGraph: DependencyGraph) -> SignalProducer<DependencyGraph, CarthageError> {
+	private func graphs(for nodes: [DependencyNode], dependencyOf: DependencyNode?,
+	                    basedOnGraph inputGraph: DependencyGraph) -> SignalProducer<DependencyGraph, CarthageError> {
```

Comment:
My preferred style is to put each argument on its own line.

```swift
func foo(
    bar: Bar,
    baz: Baz
) -> Quz {
    ...
}
```

I don’t know how the rest of the team feels about that.